next.config.js

```diff
@@ -1,17 +1,42 @@
 /** @type {import('next').NextConfig} */
+
+const APP = 'RISK';
+//const APP = 'CLIENT';
+
 const nextConfig = {
   reactStrictMode: true,
-  webpack: config => {
+  webpack: (config, { isServer } ) => {
     config.module.rules.push({
       test: /\.(graphql|gql)$/,
       exclude: /node_modules/,
       loader: 'graphql-tag/loader',
     });
+    if (!isServer) {
+      config.resolve.fallback.fs = false;
+    }
```

Comment:
Lets add very descriptive comments in our config files for future references. This is going to be a well maintained into the future